### DISCUSSION TITLE
The build status has been queued for over an hour

### DISCUSSION THREAD

**Discussion by youngminz** [codemagic-ci-cd/codemagic-docs#2723] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question

This issue is occurring in projects using the Flutter framework as well as those using the Godot framework.

### Steps to reproduce

It seems like there are no issues on the status page, but none of the projects I created are progressing with the build, and they remain in a queued status. It doesn't seem like a problem with my code, but rather an issue with Codemagic's infrastructure or possibly my account being flagged for some reason.

Steps to reproduce:
1. Trigger a build through the Codemagic website or by pushing to GitHub.
2. The build will stay in the queued status.

### Expected results

The build machine gets allocated and starts running the build pipeline.

### Actual results

The build status remains in queued for over an hour, and the build does not start.

### Build id (optional)

66dbaed83616f2f00e616a5a

**Comment by jimsakiko** [comment]
Me, too -- iOS only build. Almost 2.5 hours now. I canceled two iterations thinking it was something that I did differently.

So we wait.

**Comment by MarioC3** [comment]
I'm having the same problem. I've canceled some of the builds and restarted them, to no avail.
I started the build via the builds API.

**Comment by barees63** [comment]
Switching to an M2 instance fixed this for me

**Reply by jimsakiko** [reply]
Confirm. M2 instance is a Go. Thanks for the clue!

Now back to script troubleshooting.

**Reply by youngminz** [reply]
In my case, switching to an M2 Mac Mini results in successful builds, although I had to upgrade to a paid plan.

**Reply by Vera-Spoettl** [reply]
Can't confirm ... . Didn't help for me ...

**Comment by Balogunx** [comment]
Still Queued

**Reply by menaghaly** [reply]
same here

**Comment by mikhail-tokarev** [comment, marked as answer]
Hi all,

We've identified the issue and applied a fix. While you may see some builds starting, it will take some time to fully resolve the queue.

Sorry for the inconvenience!

**Reply by mikhail-tokarev** [reply]
All builds have now started. If the issue persists for you, please contact us. We will continue to monitor for any further issues.

**Reply by zd819** [reply]
I am getting the exact same issue right now

**Reply by jaraki** [reply]
It worked for me this morning but I also am getting this issue again

**Reply by youngminz** [reply]
I'm experiencing the same queue stuck issue again (Build ID: 66dd1229bede5fe18abafd9c). Surely, there must be better ways to handle this type of incident through automated monitoring and on-call staff. Also, I believe it would be helpful if these kinds of queue stuck issues could be promptly reflected on the status page.

**Reply by mikhail-tokarev** [reply]
Indeed, the problem reappeared. We applied a fix, and can see that builds are now starting.

We will continue investigating to ensure this doesn't happen again. Sorry again for the trouble!

**Reply by zd819** [reply]
The problem is occurring again ...

**Reply by Bobbie-Ware** [reply]
Also getting the issue again ...

**Reply by henryezeanya** [reply]
Currently having this issue.

**Comment by zd819** [comment]
Can someone look into this issue again. Not working again

**Reply by adityandar** [reply]
Same here, got queued again this morning.

**Reply by kiratheone** [reply]
same here, got queued

**Comment by Sp4Rx** [comment]
Is this happening for free accounts only?

**Comment by himesh-cm** [comment]
Hello everyone,
It should be all right now, and your builds should start soon.
Sorry for the inconvenience!

**Reply by Sp4Rx** [reply]
Currently the builds are working
